### DISCUSSION TITLE
7530 AX 7.56, Freetz bootet nach erfolgreichem Flash nicht

### DISCUSSION THREAD

**Discussion by bcsti** [Freetz-NG/freetz-ng#825] [push firmware]
Die Befehle make und push-firmware funktionieren ohne Probleme und das Image wird auch übertragen. Nach dem Reboot bootet die Fritzbox jedoch wieder in das normale FritzOS. Auch einmal zurücksetzen mit dem Recovery-Tools hat nichts gebracht.

Modell: 7530 AX, OS: 7.56, Freetz Version frisch vom 12.08.

**Comment by pwFoo** [comment]
Evtl auch ein Problem bei der 7510? #828 

Getestet mit mehreren Builds gestern und heute. Flash erfolgreich, danach gehen die Lichter aus statt einem Bootvorgang :(

**Comment by pwFoo** [comment]
Gibt es evtl eine ältere Version, die funktioniert? Wollte das "schnell" mit einer neuen Fritzbox umbauen und vorher Freetz drauf und nun funktioniert die aktuelle Version passenderweise nicht. Volltreffer zeitlich

**Comment by pwFoo** [comment]
Meine Probleme mit der 7510 habe ich nach mehrerem löschen, frischem Build und zwischendurch auch mal orig flashen lösen können, ohne einen nachvollziehbaren Fehler finden zu können.

Ich habe auch eine 7530ax, welche ich gerne mit einem minimal build flashen möchte, aber hier habe ich auch beschriebenes Bild. 
Image baut sauber, wird geflasht und am Ende stehe ich im unveränderten originalen und wie bisher konfigurierten FritzOS.

Hab zufällig gesehen, dass hier scheinbar keine Kernel Module zur Verfügung gestellt werden von AVM, was schade ist, aber damit erstmal ja nichts zu tun haben sollte. 

Hat jemand die 7530ax erfolgreich flashen können? Mit welchem Stand aus der Repo hier?
